wemake_python_styleguide/visitors/ast/functions.py

```diff
@@ -1,6 +1,6 @@
 import ast
+import typing
```

Comment:
Please, don't use `import typing`
Because typing is already verbose, extra `typing.` makes it even worse 😢

Reply:
I was bugged about it as well, but it's saying too many imported members (10 > 8)

Extra types imported by me were `Type`, `Tuple` and `Iterator`, all needed for `_get_function_exit_nodes`. I thought about using `walk.get_subnodes_by_type` instead, but that function doesn't seem to take tuples as type.

Reply:
Maybe we can extract some parts to `logic/`?